classes/order/OrderDetail.php

```diff
@@ -752,7 +752,7 @@ protected function create(Order $order, Cart $cart, $product, $id_order_state, $
         $this->product_weight = $product['id_product_attribute'] ? (float) $product['weight_attribute'] : (float) $product['weight'];
         $this->id_warehouse = $id_warehouse;
 
-        $product_quantity = (int) Product::getQuantity($this->product_id, $this->product_attribute_id, null, $cart);
+        $product_quantity = (int) Product::getQuantity($this->product_id, $this->product_attribute_id);
         $this->product_quantity_in_stock = ($product_quantity - (int) $product['cart_quantity'] < 0) ?
```

Comment:
Why is this if for ? 
```suggestion
        $this->product_quantity_in_stock = $product_quantity - (int) $product['cart_quantity'];
```

Reply:
@jf-viguier
![Výstřižek](https://user-images.githubusercontent.com/6097524/135867830-aa14e986-57cd-4a49-a367-e3aa5d4db3f2.JPG)